lessons/2-shell/slides.qmd

```diff
@@ -1,166 +1,177 @@
 
-# Why learn the shell?
+## Example file system
+
+![](file-structure-example.png)
 
--   Be able to work with tools that don't have a GUI
--   Batch large tasks (e.g. renaming files)
--   Interact with remote computers like the University HPC
+## How do you navigate your file system?
 
-# Shell in this workshop
+-   Searching
+-   Point and click
+-   "Go to path"
+
+## How do we navigate the file system *programmatically*?
+
+-   A script can't point and click!
+-   We need a system for moving around our file system through written commands.
+
+## What is the shell?
+
+-   Command-line interface, in contrast to a GUI
+-   Useful for navigating the file system
+-   ...and for automating large tasks (e.g. renaming, moving files)
+-   ...*and* for running programs that may not have a GUI
+-   ...**and** for interfacing with remote computers (e.g. the HPC)
+
+## Shell in this workshop
 
 -   git/GitHub (next 2 lessons)
 -   Navigating your file system
 -   Quarto (lesson 9)
 
-# Let's get set up!
+## Let's get set up!
 
 -   macOS comes with a shell (usually Bash or Zsh) built-in
 
 -   Windows need to install GitBash (includes Bash shell and git, which we will use later)
 
 -   Breakout rooms
 
-    -   Mac users go with Kristina
+    -   Mac users go with Eric
     -   Windows users go with Renata
 
-# Data download
+## Data download
 
 -   Download `shell-lesson-data.zip`
     -   Instructions here: <https://swcarpentry.github.io/shell-novice/index.html#download-files>
 -   Save it to your Desktop and unzip it.
 
-# Coding time
+## Coding time
 
 -   RStudio panes
 -   Set up Terminal in RStudio
 -   Navigation using Terminal and Files pane
 
-# Example file system
-
-![](file-path-example.png)
-
-# Example file system
-
-![](file-structure-example.png)
-
-# Syntax of a shell command
+## Syntax of a shell command
 
 ![](shell_syntax.png){fig-align="center"}
 
-# Coding time
+## Coding time
 
 -   Synchronize working directories between Terminal, Files Pane, and R working directory
 
-# Coding time
+## Coding time
 
 -   Use an R project to simplify working directories
 -   Copy `shell_lesson_data` to your R project
 
-# Getting help
+## Getting help
 
 ``` bash
 ls --help
 man ls
 ```
 
-# Getting help
+## Getting help
 
 ``` bash
 git --help
 ```
 
-# Getting help
+## Getting help
 
 ![](getwd.png){fig-align="center"}
 
-# Getting help
+## Getting help
 
 ![](man_page.png){fig-align="center"}
 
-# Getting help
+## Getting help
 
 ![](stack_overflow.png){fig-align="center"}
 
-# Getting help
+## Getting help
 
 ![](gpt.png){fig-align="center"}
 
-# Beware of stochastic parrots
+## Beware of stochastic parrots
 
 -   LLMs don't *understand*, but are good at mimicry
 -   Use with care:
     -   May be wrong
     -   May be inconsistent
 
-# Beware of stochastic parrots
+## Beware of stochastic parrots
 
 -   Tools like ChatGPT pose ethical issues
     -   Plagiarism
     -   Labor conditions
     -   Inputs may be sent to parent companies
 
```

Comment:
Are we including the GenAI stuff here? I thought we would move it to lesson 1 or 2 and update it a bit

Reply:
We certainly could move it! It's a little bit of a "Miscellaneous" topic; it fits everywhere and nowhere. We can decide when we meet later?